model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -242,7 +242,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install -U -q git+https://github.com/conversationai/unintended-ml-bias-analysis@1de676a31de9e43892964f71d1e38e90fc8b331e"
```

Comment:
I think this will break in the second part of the notebook. I am working on using new metrics, so let's hold on that change for now

Reply:
FWIW, the whole notebook works for me with this change.
Anyway, looking forward to your changes on using new metrics :)